### DISCUSSION TITLE
和libgrape的区别

### DISCUSSION THREAD

**Discussion by bookug** [alibaba/GraphScope#1375] [General]
graphscope中GAE就对应grape，这两边有区别么？会实时同步？

**Comment by wenyuanyu** [comment]
@bookug GAE依赖libgrape-lite，主要的增加了如下的功能：
1. 支持属性图，即有不同类型（label）的顶点，不同类型（label）的边，每种类型的点/边上有若干属性。
2. 属性图上的各种APP
3. 属性图和简单图的映射/投影
4. GAE的数据默认放在Vineyard里
5. Python和Java的支持，以及Apache Giraph的无缝Jar包兼容（on-going）
6. 点中心的编程支持
因为libgrape-lite几乎是header only的库，全是模版化，所有的组件都可以定制，以上的所有这些都直接使用libgrape-lite，并没有对libgrape-lite进行fork或者特别大的hack。
